include/Session.h

```diff
@@ -14,12 +16,30 @@
 class User;
 class Watchable;
 
+enum ActionStringsEnum{
+    createuser,
+    changeuser,
+    deleteuser,
+    dupuser,
+    listContent,
+    watchlist,
+    watch,
+    logActions, //because log exists in <vector>.
+    exitLoop // because exit exists in <unordered_map>.
```

Comment:
Isn't the convention to name enums with capital letters? so `CREATEUSER` for example?

Reply:
I though about it... ill change it... so CREATEUSER or  CREATE_USER?